terraform/example/database.tf

```
@@ -0,0 +1,13 @@
+# Teleport Database
+
+resource "teleport_database" "example" {
+    metadata {
+        name = "example"
+        description = "Test database"
+    }
+
+    spec {
+        protocol = "postgresql"
```

Teleport won't currently recognize `postgresql` as a valid protocol.
```suggestion
        protocol = "postgres"
```

---

That's interesting. I am testing everything using this definition, with no errors. Changed.